Mobile Buy SDK/Mobile Buy SDK/Operations/BUYCheckoutOperation.m

```diff
@@ -29,15 +29,15 @@
 #import "BUYClient+Internal.h"
 #import "BUYPaymentToken.h"
 #import "BUYRequestOperation.h"
+#import "BUYStatusOperation.h"
 
 @interface BUYCheckoutOperation ()
 
+@property (strong, nonatomic, readonly) BUYClient *client;
```

Comment:
On second thought, even though the operation doesn't own the client it wouldn't be a bad idea for the operation to keep a client alive if its in the process of being deallocated. If maintain only a weak reference to the client, it would then be possible that some internal operations are created and other are not (if the client is deallocated). This case would require explicit check and local strong references to the client making the code more complex. Since the strong reference doesn't (or shouldn't) unnecessarily keep the client alive in normal use cases, I would be in favour of keeping the strong reference.